Command/IndexCreateCommand.php

```diff
@@ -55,6 +61,17 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $finder->setNextFreeIndex($manager);
         }
 
+        if ($input->getOption('if-not-exists') && $manager->indexExists()) {
+            $output->writeln(
+                sprintf(
+                    '<info>Index `<comment>%s</comment>` already exists in `<comment>%s</comment>` manager.</info>',
+                    $manager->getIndexName(),
+                    $input->getOption('manager')
+                )
+            );
+            return;
```

Comment:
You are correct. As Doctrine's code about this processing, it will be 
```
return $error ? 1 : 0;
```
whatever success or error.